apis/kueue/webhooks/workload_webhook.go

```diff
@@ -65,24 +75,70 @@ func (w *WorkloadWebhook) Default(ctx context.Context, obj runtime.Object) error
 		podSet := &wl.Spec.PodSets[i]
 		setContainersDefaults(podSet.Spec.InitContainers)
 		setContainersDefaults(podSet.Spec.Containers)
+
+		if nsLimitRangesDefaults != nil {
+			setContainersDefaultsFromRR(podSet.Spec.InitContainers, nsLimitRangesDefaults)
+			setContainersDefaultsFromRR(podSet.Spec.Containers, nsLimitRangesDefaults)
+		}
 	}
 	return nil
 }
 
+func (w *WorkloadWebhook) getDefaultsFromLimitRanges(ctx context.Context, namespace string) (*corev1.ResourceRequirements, error) {
+	list := corev1.LimitRangeList{}
+	err := w.client.List(ctx, &list, &client.ListOptions{Namespace: namespace})
+	if err != nil {
+		return nil, err
+	}
+	if len(list.Items) == 0 {
+		return nil, nil
+	}
+
+	ret := &corev1.ResourceRequirements{
+		Limits:   corev1.ResourceList{},
+		Requests: corev1.ResourceList{},
+	}
+	for _, lr := range list.Items {
+		for _, lri := range lr.Spec.Limits {
+			if lri.Type == corev1.LimitTypeContainer {
+				ret.Requests = mergeResourceList(ret.Requests, lri.DefaultRequest)
```

Comment:
IIUC, `min` and `max` do not alter the values, they are just used for validation.

Reply:
yes